Tasks/GulpV0/task.json

```diff
@@ -81,9 +81,9 @@
             "name": "gulpjs",
             "type": "string",
             "label": "gulp.js location",
-            "defaultValue": "node_modules/gulp/bin/gulp.js",
-            "required": true,
-            "helpMarkDown": "gulp.js to run when agent can't find global installed gulp.  Defaults to the gulp.js under node_modules folder of the working directory.",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Alternative gulp.js to run, instead of the globally installed gulp.",
```

Comment:
I might suggest mentioning in the helpMarkDown that the input path is relative to the working directory.